src/osgEarth/Capabilities.cpp

```diff
@@ -159,10 +159,19 @@ _isCoreProfile          ( true )
     // create a graphics context so we can query OpenGL support:
     MyGraphicsContext mgc;
 
+#ifdef __ANDROID__
```

Comment:
Some comments explaining these conditionals would be helpful.

Reply:
Will do, basically we can’t currently create contexts on the native side with android so this skips that part.